resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -323,6 +323,16 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val KUBERNETES_ALLOCATION_PODSALLOCATOR =
+    ConfigBuilder("spark.kubernetes.allocation.podsallocator")
```

Comment:
Nit: Possibly this config would be better named as `spark.kubernetes.allocation.pods.allocator`? Seems that `Allocator` is used as its own word and is capitalized in class names.

I'd also update to `KUBERNETES_ALLOCATION_PODS_ALLOCATOR` if you make that change.